template/src/components/ControlledInput/index.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import {
+  InputGroup,
+  InputRightElement,
+  Input,
+  InputProps,
+  FormLabel,
+  FormControl,
+  FormErrorMessage,
+} from '@chakra-ui/react';
+import { UseFormRegisterReturn } from 'react-hook-form';
+
+interface ControlledInputProps extends InputProps {
+  label: string;
+  icon?: React.ReactNode;
+  errorMessage?: string;
+  children?: React.ReactNode;
+  register?: UseFormRegisterReturn;
+}
+
+export default function ControlledInput(props: ControlledInputProps) {
```

Comment:
This component name does not seem appropriate to me. I suggest using the name `Input` indeed, adding an alias (`Input` as `ChakraInput`) for the Chakra UI's `Input` component in order to resolve the name conflict.